openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -27,10 +27,6 @@
 // Developer styles for Staff Section
 // --------------------
 
-// edX constants
-$lighter-base-font-color: rgb(100,100,100);
-$link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.org/elements/colors/
-
 .wrapper--openassessment .wrapper--staff-area {
```

Comment:
I'd prefer not to do it as part of this story. The question though is when would we do it.... It would be good to get rid of the edx "theme" files too.